web/config/sync/core.extension.yml

```diff
@@ -11,6 +12,8 @@ module:
   link: 0
   node: 0
   page_cache: 0
+  page_manager: 0
+  page_manager_ui: 0
```

Comment:
I get your point but unlike ex `Devel` and other development tools, this only presents a UI that will be hidden for non-administrator users (I've always had the same track of thought with Views UI). But it's a good question. I don't mind having to enable it during development.